components/layout/MainNavigation.js

```diff
@@ -34,7 +34,7 @@ function MainNavigation() {
                         </Link>
                     </li>
                     <li>
-                        <Link href='/contact'>
+                        <Link href='/contact/contact'>
```

Comment:
This is not a very beautiful URL. URLs are also a means of communications to the user (and other machines) and should hierarchically represent the information architecture of the web site. Without knowing what will be on this page, I'm guessing it's contact information? Maybe put that on `/contact` (using `/contact/index.js` as the NEXT route handler).